janusgraph-solr/src/test/java/org/janusgraph/diskstorage/solr/BerkeleySolrTest.java

```diff
@@ -16,26 +16,21 @@
 
 import org.janusgraph.diskstorage.configuration.ModifiableConfiguration;
 import org.janusgraph.diskstorage.configuration.WriteConfiguration;
-import org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration;
-import static org.janusgraph.BerkeleyStorageSetup.getBerkeleyJEConfiguration;
-import static org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration.*;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
 
-/**
- * @author Matthias Broecheler (me@matthiasb.com)
- */
+import static org.janusgraph.BerkeleyStorageSetup.getBerkeleyJEConfiguration;
 
+@Testcontainers
 public class BerkeleySolrTest extends SolrJanusGraphIndexTest {
 
+    @Container
+    protected static JanusGraphSolrContainer solrContainer = new JanusGraphSolrContainer();
```

Comment:
Now that we are using test containers for Solr tests, do you want to update TESTING.md in this PR?